packages/core/src/modules/credentials/protocol/v2/__tests__/V2CredentialProtocolCred.test.ts

```diff
@@ -372,6 +374,8 @@ describe('credentialProtocol', () => {
       // then
       expect(credentialRepository.findSingleByQuery).toHaveBeenNthCalledWith(1, agentContext, {
         threadId: 'somethreadid',
+        connectionId: 'issuer',
+        role: undefined,
```

Comment:
Shouldn't the role be issuer here?

Reply:
Yeah I messed something up. I opened a new PR.